test/instrumentation/_agent.js

```diff
@@ -54,7 +49,7 @@ module.exports = function mockAgent (cb) {
     sharedInstrumentation._agent = agent
     agent._instrumentation = sharedInstrumentation
     agent._instrumentation.currentTransaction = null
-    agent._instrumentation._queue._clear()
+    agent._apmServer = mockClient(expected, cb || noop) // TODO: Expected will not work here
```

Comment:
Is this comment still accurate? What's the issue?

Reply:
Hmm I think you're right. I think this line is completely irrelevant and can just be removed. I'll update it

Reply:
Removed